app/controllers/health_profiles_controller.rb

```diff
@@ -0,0 +1,48 @@
+class HealthProfilesController < ApplicationController
+	before_action: require_login
```

Comment:
I think this is a syntax error- I think it needs to be before_action: :require_login